cmd.py

```diff
@@ -0,0 +1,15 @@
+import click
```

Comment:
it should be inside `code`
please give more meaningful name to module instead of `code`